Decred Wallet/ru-RU.lproj/Localizable.strings

```diff
@@ -139,6 +139,10 @@
 "total"  = "total:";
 "stageTimes" = "Stage Times";
 "tapToViewInformation" = "Tap to view information";
+"walletStatus" = "Wallet Status";
+"walletSynced" = "Synced";
+"walletNotSynced" = "Not Synced";
+"syncStageDescription" = "%@ - %d%%";
```

Comment:
This doesn't look translatable to me. Can it be used directly as a string literal?

Reply:
resolved in 2c74553

Reply:
yes. completely removed in 2c74553